src/psyclone/tests/psyir/nodes/array_mixin_test.py

```diff
@@ -81,33 +161,31 @@ def test_is_upper_lower_bound(fortran_reader):
     code = (
         "subroutine test()\n"
         "use some_mod\n"
-        "real a(10)\n"
+        "real a(n)\n"
```

Comment:
Please extend this test for a structure containing an array.

Reply:
I now have a separate test for an array within a structure.